core/rend/gl4/gles.cpp

```diff
@@ -661,17 +679,41 @@ static bool RenderFrame()
 	/*
 		Handle Dc to screen scaling
 	*/
```

Comment:
Does this mean RTTs are also scaled now?

Reply:
No, but they are rotated and they shouldn't. Good catch!